test/base/populators.py

```diff
@@ -24,10 +25,10 @@ def nottest(x):
 
 
 # Simple workflow that takes an input and call cat wrapper on it.
-workflow_str = resource_string(__name__, "data/test_workflow_1.ga").decode('utf-8')
+workflow_str = unicodify(resource_string(__name__, "data/test_workflow_1.ga"))
```

Comment:
I guess you'll want to convert them all, there are more in 
```
lib/galaxy/exceptions/error_codes.py
45:error_codes_json = resource_string(__name__, 'error_codes.json').decode("UTF-8")

lib/galaxy/jobs/runners/util/job_script/__init__.py
16:    resource_string(__name__, 'DEFAULT_JOB_FILE_TEMPLATE.sh').decode('UTF-8')
20:    resource_string(__name__, 'CLUSTER_SLOTS_STATEMENT.sh').decode('UTF-8')
23:    resource_string(__name__, 'MEMORY_STATEMENT.sh').decode('UTF-8')
```

Reply:
Fixed those too, thanks!